workloads/virtual-machines/overcommit.md

```diff
@@ -0,0 +1,150 @@
+# Increasing the VirtualMachineInstance Density on Nodes
+
+KubeVirt does not yet support classical Memory Overcommit Management or Memory
+Ballooning. In other words VirtualMachineInstances can't give back memory
+they have allocated. However, a few other things can be tweaked to reduce the
+memory footprint and overcommit the per-VMI memory overhead.
+
+## Remove the Graphical Devices
+
+First the safest option to reduce the memory footprint, is disabling VNC access
+to the VirtualMachineInstances, by setting
```

Comment:
I think more correctly to say `is disabling VM graphical devices` and under a side not you can say that it will make VNC not available
and can you add a small snippet for example